packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,215 @@
+
+Native:
+
+- Will set [`accessibilityRole`](https://reactnative.dev/docs/accessibility#accessibilityrole) as `alert`
+- The dismiss button will have `accessibilityLabel` as "Dismiss alert"
+
+## Usage
+
+Some example usage patterns of `Alert`.
+
+### Without title
+
+<img src="./alert-without-title.png" alt="" width="520">
+
+```jsx
+<Alert
+  intent="notice"
+  description="The payment was made 6 months ago, therefore you can’t issue refund to this merchant."
+/>
+```
+
+### Full bleed
+
+> **Note**
+>
+> Design update pending to remove all borders and border radii
+
+This full bleed layout works for all desktop, mobile and native when the `isBorderless` prop is passed
+
+<img src="./alert-mobile-full-bleed.png" alt="" width="520">
+
+```jsx
+<Alert
+  title="International Payments Only"
+  isBorderless
+  description={
+    <>
+      Use vendor payouts to quickly generate invoices.{' '}
+      <Link href="https://razorpay.com">Know More</Link>
+    </>
+  }
+/>
+```
+
+## Open questions
+
+**Q1.** Should we use a different intent naming convention in code like `success`, `warn`, `info`, `error`? Currently this is same as design
+
+**A1.** It makes more sense to keep the terminologies similar in both design and code (because designs are what gets translated into code). [Related discussion](https://github.com/razorpay/blade/pull/573#discussion_r929998203).
+
+**Q2.** It's possible to pass any JSX as content but ideally we want to restrict it to few components (like `Link`, `string`, `List`). Doing these on code side would mean lot of checks. Similarly, restriction on `SecondaryAction` which should always be used with `PrimaryAction`. How do we do these - TS / runtime checks, etc.? One simpler alternative is to rely on design side restrictions for these but it makes the API more flexible should someone want to try things different from design.
+
+**A2.** We can do some static checks with TypeScript where feasible, for example enforcing `SecondaryAction` to be used only with `PrimaryAction` but some checks are outside scope of TypeScript and in the realm of runtime checks which at the moment seems like an overkill, for example restricting the content type for `description` prop.
+
+**Q3.** Should we make alerts focussable so they're discoverable by tabbing? It would mean also showing a visible keyboard focus indicator. Not doing this currently because I can't think of possible usecases right now, maybe we can revisit this later if / when needed.
+
+**A3.** TBD
+
+**Q4.** Whether to use `intent` or `variant`?
+
+**A4.** The rationale to use `intent` is because it signifies what the alert is meant for (success, error et al.). Also, we might need to extend alerts later to have different _variants_ like outline, ghost, etc. for which having a `variants` prop reserved for later would make more sense. There are some alternate terminology instead of `intent` that could be considered for example `colorScheme`.
+
```

Comment:
One more open question from the Design side is how we will render the links inside the `description` prop (in JSX)?

Reply:
@kamleshchandnani @divyanshu013 Shall we remove links inside the description for now? Until we have a live use case?
// @nashcheez

Reply:
We can pass `Link` component as part of the JSX for that in code. Hope that's what you meant:

```jsx
<Alert
  description={
    <>
      Currently you can only accept payments in international currencies using PayPal. You cannot
      accept payments in INR (₹) using PayPal. <Link href="https://razorpay.com">Know More</Link>
    </>
  }
  ...
/>;
```

- [More details in the doc](https://github.com/razorpay/blade/blob/div/alerts-decisions/packages/blade/src/components/Alert/_decisions/decisions.md#api)

Reply:
Good reminder @saurav12! 
@divyanshu013 links inside the description in the high contrast version are one thing we haven't solved yet design-wise, let's add it to the open questions and close this.

Reply:
Added in https://github.com/razorpay/blade/pull/573/commits/008e599e1ff67b63f5094b85c0d43e83e440f74b